pyproj/crs/__init__.py

```diff
@@ -24,4 +24,24 @@
     ProjectedCRS,
     VerticalCRS,
 )
-from pyproj.exceptions import CRSError  # noqa: F401  pylint: disable=unused-import
+from pyproj.exceptions import CRSError
+
+__all__ = [
+    "CoordinateOperation",
```

Comment:
CoordinateOperation, Datum, Ellipsoid, PrimeMeridian and here for backwards compatibility. I don't recommend adding them to `__all__`.